src/reducers/versions.spec.tsx

```diff
@@ -616,4 +619,237 @@ describe(__filename, () => {
       });
     });
   });
+
+  describe('createInternalDiffs', () => {
```

Comment:
I like the approach you took with this, deferring the tests for more "nested" data for a test to be executed later. I think what we've done in the past is created helper methods like `createInternalChange` and `createInternalHunk` and then used those in the tests for assertions, and then we've also had to include tests for the helper methods themselves. This (what you've done here) seems like a reasonable approach to me, but I thought I'd point out that it's different from what we've done in the past.